lib/sda/export_test.go

```diff
@@ -47,3 +47,21 @@ func (o *Overlay) TokenToNode(tok *Token) (*TreeNodeInstance, bool) {
 	tni, ok := o.instances[tok.Id()]
 	return tni, ok
 }
+
+func (h *Host) AbortConnections() error {
+	for _, c := range h.connections {
+		h.unregisterConnection(c)
+	}
+	close(h.ProcessMessagesQuit)
+	return h.host.Close()
+}
+
+func (h *Host) CloseConnections() error {
```

Comment:
CloseConnection should it not unregister the connection too ? Since it's closed we won't use it anymore ..